### DISCUSSION TITLE
What does upnplib have to do with Raspberry Pi?

### DISCUSSION THREAD

**Discussion by seamusdemora** [upnplib/upnplib#10] [General]
Hello Ingo! 

Long time no hear from!   Apologies for contacting you this way, but I wasn't sure how best to reach you. I guess this project is keeping you busy?  I think RPi SE will die soon due to discontinuation of production of hardware & perhaps other changes. One thing is that the new users aren't getting any smarter :) lol

**Comment by ingo-h** [comment]
@seamusdemora 
Hi Seamus, I am very pleased to hear from you :-)
Yes, this project keeps me very busy. I like to use RPis as media renderer and control points using UPnP but found that the old pupnp library started by Intel is outdated and only poorly supports ipv6. So i decided to reengineer it.

I'm not up to date with Raspberry Pi but will have a look at it again next time. What is RPi SE and what is the story behind it? As far as I can see there is still no successor of the Raspberry Pi 4 Model B. Do you mean the hardware production will be discontinued in general?

**Comment by seamusdemora** [comment]
@ingo-h 

Oh - RPi SE is just shorthand for Raspberry Pi Stack Exchange - the same site we collaborated on. As far as RPi production, all I know for certain is that it has been impossible to buy a RPi for many months now - they aren't available anywhere that I can find. I also read something recently that they (the manufacturers of RPi) have re-started manufacture, but all the capacity is going to their commercial customers. So - who knows what will happen? But at some point, if they're not making boards people can buy, there will be little point in RPi SE. So - it's good you have found a full-time project. 

I experimented with a small micro-controller board from Microchip a few months ago, but it's not going well :(  Their software makes no sense to me, and several of their "demo" programs didn't work at all. I wrote some uC code years ago (C language for 8051), but today's Microchip IDE is incomprehensible to me. 

I'm out of time now, so I'll have to close. Stay well, and let's keep in touch.
